Tests/FoundationEssentialsTests/ProcessInfoTests.swift

```diff
@@ -99,8 +99,7 @@ final class ProcessInfoTests : XCTestCase {
         XCTAssertTrue(
             now > 1, "ProcessInfo returned an unrealistically low system uptime")
         // Sleep for 0.1s
-        var ts: timespec = timespec(tv_sec: 0, tv_nsec: 100000000)
-        nanosleep(&ts, nil)
+        Thread.sleep(forTimeInterval: 0.1)
```

Comment:
`Thread` comes from the `Foundation` module which we'd like to try to avoid in this project (even in the tests). Is there something else we could use here, maybe some `async`/`await` or `Task` related API to accomplish the same goal in a portable manner?

Reply:
https://developer.apple.com/documentation/swift/task/sleep(for:tolerance:clock:) should suffice I suppose for that.